layouts/partials/site-head.html

```diff
@@ -63,18 +63,6 @@
 </script>
 {{ end }}
 
-{{- if .Site.Params.recaptchaKey -}}
```

Comment:
This code has been moved from the `site-head` partial to the `site-scripts` partial. Since it doesn't need to be run in the `head` element, it's best to move this down with the scripts so that it doesn't block HTML from rendering. It's not really noticeable if a user has a good connection, but when they don't it can be one of the reasons that leads to slow page loads.